draft-ietf-mops-streaming-opcons.md

```diff
@@ -375,22 +380,28 @@ then chooses a bitrate for each of the video, audio, subtitles and metadata (amo
 typically adjusting as changes in available bandwidth occur in
 the network or changes in capabilities occur during the playback (such as available memory, CPU, display size, etc.).
 
-The choice of bitrate occurs within the context of optimizing for
-some metric monitored by the client, such as highest achievable
-video quality or lowest chances for a rebuffering (playback stall).
+##Adaptive Encoding {#adapt-encode}
 
-##Segmented Delivery
+Media servers can provide media streams at various bitrates because the media has been encoded at various bitrates. This is a so-called "ladder" of bitrates, that can be offered to media players as part of the manifest that describes the media being requested by the media player, so that the media player can select among the available bitrate choices.
+
```

Comment:
The media server may also choose to alter which bitrates are made available to players by adding or removing bitrate options from the ladder delivered to the player in subsequent manifests built and sent to the player.   

In this way both the player, through its selection of bitrate to request from the manifest, and the server, through its construction of the bitrates offered in the manifest, are able to affect the network utilization.

Reply:
I used this text with like, 2 editorial changes. Thanks for your help on this!